docs/core-concepts/versioning.md

```diff
@@ -1,6 +1,202 @@
+
+Versioning Form
+---------------
+
+MAJOR.MINOR.PATCH[-PRERELEASE.BUILDNUMBER]
+
+Decision Tree
+-------------
+
+MAJOR when:
+  - drop support for a platform
+  - adopt a newer MAJOR version of an existing dependency 
+  - turn a quirk off by default
+
+MINOR when:
+  - add public API surface area 
+  - add new behavior
+  - adopt a newer MINOR version of an existing dependency
+  - introduce a new dependency 
+  
+PATCH when:
+  - make bug fixes
+  - add support for a newer platform
+  - adopt a newer PATCH version of an existing dependency
+  - anything other change
+
+When determining what to increment when there are multiple changes, choose the highest kind of change.
+
+Versioning Scheme
+=================
+
+.NET Core can be defined as and will version in the following way:
+
+- A runtime and framework implementation, distributed as packages. Each package is versioned independently.
+- A set of meta-package packages that reference and make the fine-grained packages easier to reference and use together as a versioned unit. Each meta-package has its own versioning scheme.
+- A set of target frameworks (e.g. netstandard) that represent a progressively larger API, described in a set of versioned snapshots.
+
+Packages
+--------
+
+Library packages evolve and version independently. Packages that overlap with .NET Framework System.\* assemblies typically use 4.x versions, aligning with .NET Framework 4.x versioning (a historical choice). Packages that do not overlap (e.g. System.Metadata) with .NET Framework libraries typically start at 1.0 and increment from there.
+
+The lower-level packages (most of the ones with 4.x versions) are treated specially due to being at the base of the platform. Explanation:
+
+- The lower-level packages will typically version as a set, since they have implementation-level dependencies between them. 
+- APIs will typically only be added to the lower-level packages as part of a minor or major .NET Core release since adding APIs would require adding a new netstandard version, which currently requires an update to NuGet. As a result, these packages will only be updated with patch versions (no new APIs) between .NET Core releases.
+
+Meta-Packages
+-------------
+
+Versioning for .NET Core meta-packages is based on the target framework that they map to. The meta-packages adopt the version number of the highest version (e.g. netstandard1.5) of the target framework it maps to in its package closure. The patch version for the meta-package is used to represent updates to the meta-package to reference updated packages (that don't include a new target framework version). As a result, the meta-packages are not strictly SemVer compliant because their versioning scheme doesn't represent the degree of change in the underlying packages, but only API level. 
+
+There are two primary meta-packages for .NET Core.
+
+**NETStandard.Library**
+
+- v1.5 as of .NET Core 1.0.
+- Maps to `netstandard` TFM. 
```

Comment:
Explain what TFM is. Remember, even us new people to the team had no idea what this TLA means.